shared/settings/nav/index.stories.tsx

```diff
@@ -4,6 +4,7 @@ import {action, storiesOf} from '../../stories/storybook'
 
 const defaultProps = {
   badgeNumbers: {},
+  contactsLabel: 'Import contacts',
```

Comment:
Cap C

Reply:
First letter only capitalized is actually more consistent with how we title most of the menu items / modal titles / buttons in the app. The "Your account" change was at Cécile's request (see that title on desktop in master)

Reply:
👌 thanks